EntityFramework.Exceptions.Tests/OracleTests.cs

```diff
@@ -7,17 +9,46 @@ namespace EntityFramework.Exceptions.Tests
 {
     class OracleTests : DatabaseTests, IClassFixture<OracleTestContextFixture>
     {
-        public OracleTests(OracleTestContextFixture fixture) : base(fixture.Context)
+        OracleTests(OracleTestContextFixture fixture) : base(fixture.Context)
         {
         }
+
+        private async Task DeleteOfProductInSaleShouldThrowReferenceConstraintException()
```

Comment:
I think we need this test for other databases too so I guess it's better to have this test in the base class too